### DISCUSSION TITLE
Does the Firmware Management (Port 203) is supported by LoRaMac-Node ?

### DISCUSSION THREAD

**Discussion by Kasimashi** [Lora-net/LoRaMac-node#1604] [Q&A]
Hello Everyone, 

Does the Firmware Management (Port 203) is supported by LoRaMac-Node ?
I can't find any code related with this LoRaWAN feature ? 

Cf : https://resources.lora-alliance.org/technical-specifications/ts006-1-0-0-firmware-management-protocol

BR

**Comment by Kasimashi** [comment]
No news ?

**Comment by mluis1** [comment]
Sorry for the late answer.

Currently it is not supported and unfortunately I cannot tell you when it will be.
